public/client.js

```diff
@@ -92,45 +102,57 @@ socket.on('user join', function(data) {
 socket.on('Display Message', function(data) {
     var today = new Date();
     var class_name;
-    if(socket.username == data.user) {
+    if (socket.username == data.user) {
         class_name = 'self';
-    }
-    else {
+    } else {
         class_name = 'others'
     }
     // Adding Emoji
     var p = data.msg;
     var colon1 = p.indexOf(":");
     while (colon1 != -1) {
-      var colon2 = p.indexOf(":", colon1 + 1);
-      if (colon2 != -1) {
-        emoji_name = p.slice(colon1+1, colon2);
-        position = emoji_names.indexOf(emoji_name)
-        if (position != -1) {
-          p = p.slice(0, colon1) + "<img class=\"emoji\" src=\"images/emoji/" + emojis[position] + ".png\">" + p.slice(colon2 + 1);
+        var colon2 = p.indexOf(":", colon1 + 1);
+        if (colon2 != -1) {
+            emoji_name = p.slice(colon1 + 1, colon2);
+            position = emoji_names.indexOf(emoji_name)
+            if (position != -1) {
+                p = p.slice(0, colon1) + "<img class=\"emoji\" src=\"images/emoji/" + emojis[position] + ".png\">" + p.slice(colon2 + 1);
+            }
+            colon1 = p.indexOf(":", colon2 + 1);
+        } else {
+            break;
         }
-        colon1 = p.indexOf(":", colon2+1);
-      }
-      else {
-        break;
-      }
     }
-    p = "<p>" + p + "</p>";
-    var room_id = convertIntoId(data.room);
-    if(class_name == 'self' || ($("#" + room_id + "-msg").children(".chat")[0].scrollHeight - $("#" + room_id + "-msg").children(".chat").scrollTop() == scrollDiff)){
-        $("#"+ room_id +"-msg").children(".chat[data-chat='person1']").append("<div class='bubble " + class_name + "' data-chat ='person1'>\
-                                " + p + "<br>\
-                                <span class='info'>" + data.user + "</span>\
-                           </div>");
-        var room_id = convertIntoId($(".active").attr("id"));
-        var height = $("#" + room_id + "-msg").children(".chat")[0].scrollHeight;
-        $("#" + room_id + "-msg").children(".chat").scrollTop(height);
+    var dateTime = new Date();
+    var hours = dateTime.getHours().toString(10);
+    var mins = dateTime.getMinutes().toString(10);
+    if (hours.length == 1) {
+        hours = '0' + hours;
+    }
+    if (mins.length == 1) {
+        mins = '0' + mins;
     }
-    else{$("#"+ room_id +"-msg").children(".chat[data-chat='person1']").append("<div class='bubble " + class_name + "' data-chat ='person1'>\
-                                " + p + "<br>\
-                                <span class='info'>" + data.user + "</span>\
-                           </div>");
+    // Format Message
+    const div = document.createElement('div');
+    const username = document.createElement('small');
+    const TimeStamp = document.createElement('small');
```

Comment:
Can you change `TimeStamp` to `timestamp` ? I think that is fine. CapWords in JS looks weird, probably suited in Python.

Reply:
Done!